### PR TITLE
Add semantic header tag and make CSS adjustments to submit button

### PR DESCRIPTION
**Pull request by npydi**
This PR contains: 
Changes to <header> elements and CSS adjustments to submit button.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -37,15 +35,17 @@ label {
     margin-left: auto;
     margin-right: auto;
     margin-bottom: 1.5em;
+    margin-top: 0.2em;
 }
 
 #submit {
```
Comment (npydi) [inline css/style.css:38]: ◻️ kindly add the button styling: Active, Hover, and Pressed as mentioned in Figma.
image

File: index.html [hunk 2]
```diff
@@ -9,8 +9,10 @@
...
     <!-- <img src="images/purple.jpeg" alt="purple"> -->
-    <h1 id="title">Microverse Trail Survey Form</h1>
-    <p id="description"><i>Thank you for taking the time to help us improve the platform</i></p>
+    <header>
+      <h1 id="title">Microverse Trail Survey Form</h1>
+      <p id="description"><i>Thank you for taking the time to help us improve the platform</i></p>
+    </header>
 
     <section id="sec">
       <form id="survey-form">
```
Comment (npydi) [inline index.html:18]: Your website looks really awesome👏. As **in-line styles Are Not Best Practice** kindly remove the in-line styles applied to the elements inside the section Inline. 
*In-line styles might give you more granular control, but it also makes other aspects of design and development such as consistency more difficult.*

**Review by npydi — COMMENTED** [review]
This is an image
- Good job on completing the Survey-form Microverse project
However there are a few minor changes, please follow the comments below to make the required changes

## Highlights
- Followed and completed all the instructions :100:
- All required tests passed for the project :100:

## Changes required ♻️ 
◻️ It's good to have a Readme file for every project, going forward an excellent and professional readme is a requirement for approval.
◻️ Make sure to add linter checks

*Check comments under the review*

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @npydi 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Required Changes:
- [ ] It's better if the comments are phrased politely, as suggestions instead of orders. For example:
  - This is a polite suggestion: "I think it would be better to use semantic tags"
  - This reads as an order: "Use semantic tags"
- [ ] The comments should always include an explanation, a "why", for each requested change. For example:
  - * This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
- [ ] Side note: when you create inline comments, kindly don't submit the review just yet until you write the summary then submit the review once and for all. This way your review will be organized and professional ✨ 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by npydi** [comment]
Hi @ha-manel, thank you for reviewing my code and giving helpful suggestions. 
- I modified my comments as per your mentions.
- I will make sure not to submit my in-line comments individually going forward from my next code reviews.

**Review by sajid-munawar — COMMENTED** [review]
### Hi there,
Great job on this project review! 👍🏻
# Status: Approved ✔️
Excellent job on this project review. Keep it up 👏
Your project review is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
>
### To Highlight
- [x]  Added descriptive inline comments. :heavy_check_mark: 
- [x]  Review comments are polite, use suggestions and give reasons for requested change :heavy_check_mark:
- [x] Made use of [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) :heavy_check_mark: 
>
 ## [Optional] suggestions:
- [x] Made use of emojis in project review summary to make the code review more human-friendly 👽🤖. :heavy_check_mark: 
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**